### PR TITLE
[[ Bug 14554 ]] Add user guides for widgets

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: extensions/widgets/browser/docs/guide/guide.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Alternatively it can be created in script using:
+
+	create widget as "com.livecode.widget.browser"
+
+## Displaying content in a browser
+
+The most straightforward way to use the browser widget is to set its 
+`url` property to display a web page. It can also be set to a local file 
+url, for example to display a PDF.
```
Comment (montegoulding) [inline extensions/widgets/browser/docs/guide/guide.md]: PDF is probably not the best suggestion as its not supported on all platforms

File: extensions/widgets/browser/docs/guide/guide.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Alternatively the `htmlText` property can be used to render specific 
+chunks of html within the browser object. 
+
+## Controlling navigation
+The browser sends the following messages which can be used to control navigation within the browser. See the API documentation for more details:
+* browserDocumentLoadBegin
```
Comment (montegoulding) [inline extensions/widgets/browser/docs/guide/guide.md]: add a line above the list here

File: extensions/widgets/clock/docs/guide/guide.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Clock Widget
+The clock widget displays the current time in a given time zone, using
+an analog clock face.
+
+![Clock widget](images/clock.png)
+
+## Creating a Clock
+A Clock widget can be created by dragging it out from the Tools
```
Comment (montegoulding) [inline extensions/widgets/clock/docs/guide/guide.md]: We probably need to decide if we want to capitalise the widget name throughout. I think this is an issue for most of the guides.

File: extensions/widgets/graph/docs/guide/guide.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+just show markers:
+
+	local tMarkers
+	put "filled circle" & return & "filled diamond" into tMarkers
+	set the markerStyles of widget "Line Graph" to tMarkers
+	set the showLines of widget "Line Graph" to false
+
+![Line graph with no grid or markers](images/data-no-lines.png)
+	
+This is helpful for creating a scatter plot.
```
Comment (montegoulding) [inline extensions/widgets/graph/docs/guide/guide.md]: Due to the treatment of x axis as labels we can't use the widget as a scatter plot at the moment so perhaps delete this line?

File: extensions/widgets/graph/docs/guide/guide.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+The final property of the Line Graph is the `hilitedCoordinates` 
+property. You can set the `hilitedCoordinates` to a point on any of the 
+lines on the chart. For example:
+
+	local tHilitedDataPoint
+	put "Jul,18" into tHilitedDataPoint
+	set the hilitedCoordinates of widget "Line Graph" to tHilitedDataPoint
+
+![Line graph with hilited coordinates](images/data-hilited-coordinates.png)
```
Comment (montegoulding) [inline extensions/widgets/graph/docs/guide/guide.md:89]: Ohh... optimal month or @montegoulding to visit Scotland ;-)

File: extensions/widgets/clock/docs/guide/guide.md (hunk 3 above)
Comment (livecodeali) [reply extensions/widgets/clock/docs/guide/guide.md]: Good point... It looks like in most of the other guides, title case for section headings is standard. So I think this is one of the few correct cases.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok b337118

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok b33711826d94257360e71cf31850964d558b7894

**Comment by livecode-vulcan** [comment]
😎 test success b33711826d94257360e71cf31850964d558b7894

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1379)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1395)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1369)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/322)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/320)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1373)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1399)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1379)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1393)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1378)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1388)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1429)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1369)

**Pull request merged**